app/jobRoleData.js

```diff
@@ -0,0 +1,12 @@
+const axios = require('axios')
+
+exports.getJobRoles = async () => { 
+      try {  
+        const jobRoleResponse = await axios.get('http://localhost:8080/api/job-roles')
+        return jobRoleResponse.data
+      } catch (e) {
+         return new Error('Could not get job roles')
```

Comment:
Does this exception contain a message we can also add to the error message? The more detailed we can be the better